frontend/src/components/common/Verortung.vue

```diff
@@ -81,15 +81,22 @@ import {
   VerortungDto,
 } from "@/api/api-client/isi-backend";
 import SaveLeaveMixin from "@/mixins/SaveLeaveMixin";
+import AbfrageSecurityMixin from "@/mixins/security/AbfrageSecurityMixin";
 
 @Component({
   components: { CityMap },
 })
-export default class Verortung extends Mixins(GeodataEaiApiRequestMixin, SaveLeaveMixin) {
+export default class Verortung extends Mixins(GeodataEaiApiRequestMixin, SaveLeaveMixin, AbfrageSecurityMixin) {
   private verortungCardTitle = "Verortung";
 
   @VModel({ type: VerortungModel }) verortungModel?: VerortungModel;
 
+  @Prop({ type: Boolean, default: false })
```

Comment:
Meiner Meinung nach wäre eine andere Implementierung wartbarer. Ich habe die "Bedeutung" dieser Properties erst spät erkannt. Was hier gemeint ist, ist der Context in dem die Komponente verwendet wird. Vorschlag:

`export const enum VerortungContext {
  UNDEFINED,
  ABFRAGE,
  BAUVORHABEN
}`

@Prop({ type: VerortungContext , default: UNDEFINED})
private readonly context!: VerortungContext ;

get isEditable(): boolean {
    if (this.context === VerortungContext.ABFRAGE) {
        return this.isEditableByAbfrageerstellung;
     } else {
     if (this.context === VerortungContext.BAUVORHABEN) {
        return this.isEditableBySachbearbeitung;
     }
    return false;
}

Reply:
ja, stimmt. Dass ist eine bessere Lösuung.